src/gui/FileBrowser.cpp

```diff
@@ -753,6 +770,14 @@ QList<QAction*> FileBrowserTreeWidget::getContextActions(FileItem* file, bool so
 		result.append(toSampleTrack);
 	}
 
+	if (fileIsSample)
+	{
+		auto openInSlicer = new QAction(tr("Send to new SlicerT instance"), nullptr);
```

Comment:
```suggestion
		auto openInSlicer = new QAction(tr("Send to new SlicerT instance"));
```

The second parameter of this overload has a default value of `nullptr` so it can be omitted. [Source (qt docs)](https://doc.qt.io/archives/qt-5.15/qaction.html#QAction-1)

Reply:
[Also true for Qt6](https://doc.qt.io/qt-6/qaction.html#QAction-1)!